src/runtime/runtime.ts

```diff
@@ -252,42 +224,26 @@ export abstract class Runtime {
     );
   }
 
-  private makeSubAccount(name: string): string {
-    return `${name}.${this.masterAccount}`;
-  }
-
-  async createAccount(name: string, keyPair?: nearAPI.utils.key_pair.KeyPair): Promise<Account> {
-    const accountId = this.makeSubAccount(name);
-    const pubKey = await this.addKey(accountId, keyPair);
-    await this.root.najAccount.createAccount(
-      accountId,
-      pubKey,
-      new BN(this.config.initialBalance!)
-    )
-    const account = this.getAccount(name);
-    this.accountsCreated.set(accountId, name);
-    return account;
-  }
-
-  async createAndDeploy(
+  async createAccount(
     name: string,
-    wasm: string,
+    {
+      keyPair,
+      initialBalance = this.config.initialBalance!,
+    }: { keyPair?: KeyPair; initialBalance?: string } = {}
   ): Promise<Account> {
-    const account = await this.createAccount(name);
-    const contractData = await fs.readFile(wasm);
-    const result = await account.najAccount.deployContract(contractData);
-    debug(`deployed contract ${wasm} to account ${name} with result ${JSON.stringify(result)} `);
-    this.accountsCreated.set(account.accountId, name);
-    return account;
+    return this.root.createAccount(name, {keyPair, initialBalance});
+  }
+
+  async createAndDeploy(name: string, wasm: string | Buffer): Promise<Account> {
+    return this.root.createAndDeploy(name, wasm);
   }
```

Comment:
Should we delete these methods from `runtime` entirely?

Reply:
Yeah good idea.